src/Directory.Build.targets

```diff
@@ -8,4 +8,31 @@
   <Import Project="packaging.stubs.targets" Condition="'$(MSBuildProjectExtension)' != '.pkgproj'" />
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.targets))\Directory.Build.targets" />
+
+  <!--
+    Package build can be skipped by invoking PackIfServiced instead of Pack targets.
```

Comment:
Was this comment line meant to be on this target, or `PackIfServiced`? Maybe `PackIfServiced` should be defined first since it's a better introduction to what we're trying to accomplish with this set of targets?

Reply:
Thanks - I'll reorder targets.